dotcom-rendering/src/web/components/FrontCard.tsx

```diff
@@ -4,7 +4,26 @@ import { Card } from './Card/Card';
 
 type Props = {
 	trail: TrailType;
-} & Partial<CardProps>;
+	headlineSize?: SmallHeadlineSize;
+	showQuotes?: boolean;
+	showByline?: boolean;
+	imageUrl?: string;
+	imagePosition?: ImagePositionType;
+	imagePositionOnMobile?: ImagePositionType;
+	imageSize?: ImageSizeType;
+	trailText?: string;
+	avatar?: AvatarType;
```

Comment:
I think this can be removed. We only show this for certain `format`s and only if it is present